Documentation/ColumnsConfig/Type/Category/Properties/ForeignTablePrefix.rst

```diff
@@ -5,8 +5,8 @@
 foreign_table_prefix
 ====================
 
-..  confval:: foreign_table_prefix (type => category)
-
+..  confval:: foreign_table_prefix
+    :name: category-properties-foreign-table-prefix
```

Comment:
```suggestion
    :name: category-fforeign-table-prefix
```

Above you use "category-foreign-table-item-group", here you add "properties". Please be consistent. I suggest to remove "properties"

Reply:
Copy & Pasted that accidently